definitions/npm/react-redux_v5.x.x/flow_v0.89.x-/react-redux_v5.x.x.js

```diff
@@ -52,6 +52,8 @@ declare module "react-redux" {
     // and provide the StateProps type to the SP type parameter.
     | ((state: S, ownProps: OP) => (state: S, ownProps: OP) => SP);
 
+  declare type Bind<D> = <A, R>((...A) => R) => (...A) => $Call<D, R>;
```

Comment:
This is what [`bindActionCreators()`](https://redux.js.org/api/bindactioncreators) actually does.

Reply:
Should the corresponding change be made to [redux libdef](https://github.com/flow-typed/flow-typed/blob/master/definitions/npm/redux_v3.x.x/flow_v0.55.x-/redux_v3.x.x.js#L50-L51) as well?